.github/ISSUE_TEMPLATE/g3_bug.yml

```diff
@@ -0,0 +1,132 @@
+name: Report a bug
+description: |
+  You found a bug in Flutter causing your application to crash or
+  throw an exception, a widget is buggy, or something looks wrong.
+labels: 'from: g3'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Flutter!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on Stack Overflow:
+
+          - https://flutter.dev/
+          - https://api.flutter.dev/
+          - https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+  - type: markdown
+    attributes:
+      value: |
+        Before filling the form fields, please consider the following:
+        - Ensure that you have searched the [existing issues](https://github.com/flutter/flutter/issues)
+        - Read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)
+- type: checkboxes
+  id: priority-indicator
+  attributes:
+    label: Help us understand the severity of this issue
+    description: Please only choose one
+    options:
+      - label: causing severe production issues e.g. malfunctions or data loss
+      - label: blocking next binary release
+      - label: blocking a client feature launch within a quarter
+      - label: nice-to-have but does not block a launch within the next quarter
+  - type: textarea
```

Comment:
Would it make sense to include a place to write the b/ bug number?

Reply:
I thought we agreed that they wouldn't make a b/ bug version... and would only file github ones from now on.

Reply:
Yes, you're right =)